examples/clojure_test_client/src/khan_client/web.clj

```diff
@@ -0,0 +1,38 @@
+(ns khan-client.web
+  (:require [compojure.core :refer [defroutes GET PUT POST DELETE ANY]]
+            [compojure.handler :refer [site]]
+            [compojure.route :as route]
+            [clojure.java.io :as io]
+            [ring.adapter.jetty :as jetty]
+            [ring.util.response :refer [redirect]]
+            [environ.core :refer [env]]
+            [khan-client.api :as khan]
+            [clojure.java.browse :as b]
+            [clj-http.client :as http]))
+
+(defn khan-api []
+  (khan/request-token-handler khan/consumer "http://localhost:5000/auth"))
+
+(defn khan-auth [token]
+  {:status 200                                                                                         
+   :headers {"Content-Type" "text/html"}                                                                  
+   :body (str token)})
+
+(def access-token-response (atom {}))
+(def credentials (atom {}))
+
+(defroutes app
+  (GET "/" []
+       (khan-api))
+  (GET "/auth" {params :params}
+       (swap! access-token-response conj (khan/access-token-request params))
+       (swap! credentials conj (khan/credential-handler @access-token-response 
+                                                        {:email "suma_coach@seatacunited.org"}))
+       (http/get "https://www.khanacademy.org/api/v1/user?email=suma_coach@seatacunited.org" 
```

Comment:
Looks like you have your email address here. Can you change it to be more generic? I think `/api/v1/user` defaults to the current user, so you could just leave it out. If you want to demonstrate how to pass parameters, you could pass `format=pretty` or use some other endpoint.